components/claim/create-reward-claim.tsx

```diff
@@ -45,15 +45,14 @@ const CreateRewardClaim = () => {
   const [loading, setLoading] = useState(false);
   const [claimDeadlineAsString, setClaimDeadlineAsString] = useState("");
   const { showCustomErrorToast, showSuccessToast } = useCustomToasts();
-  // ester: if the user has not selected fromDate, toDate, or claimDeadline we should display an error message
-  // amount is taken from contract thus not editable
   const [claimInput, setClaimInput] = useState<Claim>({
     fromDate: null,
     toDate: null,
     claimDeadline: null,
     amount: ""
   });
   const [minDate, setMinDate] = useState<Date | null>(null);
+  const ONE_WEEK_MILLISECONDS = 604800000;
```

Comment:
this period is set by the contract function `minimumClaimDeadline` we shouldn't hardcode it